statistics-rest/src/main/resources/application.properties

```diff
@@ -0,0 +1,12 @@
+#JPA
+spring.jpa.database=POSTGRESQL
+spring.datasource.platform=postgres
+spring.jpa.hibernate.ddl-auto=update
+spring.datasource.driver-class-name=org.postgresql.Driver
+serviceInfoPath=classpath:service-info.json
+
+
+#database
+spring.datasource.url=jdbc:postgresql://pgsql-hlvm-024.ebi.ac.uk:5432/unpstspro?currentSchema=uniprot
+spring.datasource.username=uniprot
```

Comment:
We should not commit any internal server user/passwd

Reply:
removed.